foundationdb/assets/monitors/read_latency_probe.json

```diff
@@ -2,11 +2,11 @@
   "version": 2,
   "created_at": "2022-03-14",
   "last_updated_at": "2022-05-25",
-  "title": "FoundationDB Read Latency Probe",
+  "title": "Read operation took a long time",
   "tags": [
     "integration:foundationdb"
   ],
-  "description": "Get notified when a read operation performed on the FoundationDB cluster to measure current latency took a long time.",
+  "description": "Read latency refers to the time it takes for read operations to be completed within a database system. This monitor tracks the average read latency for various operations within FoundationDB to avoid issue related to data retrieval.",
```

Comment:
```suggestion
  "description": "Read latency refers to the time it takes for read operations to be completed within a database system. This monitor tracks the average read latency for various operations within FoundationDB to avoid issues related to data retrieval.",
```